packages/backend/src/open_payments/wallet_address/routes.ts

```diff
@@ -60,8 +61,8 @@ export async function getWalletAddress(
     )
 
   ctx.body = walletAddress.toOpenPaymentsType({
-    authServer: deps.config.authServerGrantUrl,
-    resourceServer: deps.config.openPaymentsUrl
+    authServer: `${ensureTrailingSlash(deps.config.authServerGrantUrl)}${walletAddress.tenantId}`,
```

Comment:
can we avoid the need to have `urlWithoutTenantId` if we don't include the `tenantId` in the `authServer` response for now? We can just add it in once the AS is ready. Might be less work overall

Reply:
Ah, I did it the other way around, so that we only need to look for the `urlWithoutTenantId` usages? Can we leave it like this? 😬